.gitlab-ci.yml

```diff
@@ -2,7 +2,17 @@ include:
   remote: 'https://raw.githubusercontent.com/ebi-ait/gitlab-ci-templates/master/build-release-deploy.yml'
 
 Test:
-  image: quay.io/ebi-ait/ingest-base-images:trion_ng-cli-karma_11.2.6
+  image: quay.io/ebi-ait/ingest-base-images:node_14.15.5-alpine
```

Comment:
Can we use an environment variable here that would take the value from the `package.json` or `.nvmrc` file or at least have the gitlab job fail if there is a version mismatch? I think yarn/npm will fail if the node engine does not match the *engine* configuration in `package.json`.

Reply:
I agree that a single source of truth would be better but I can't think of a way we could do that.

Also due to our use of quay to store base images (as well as other factors) changing the angular/node version is a big deal that we shouldn't need to do that often, so I think it's okay that we have to change the config in a few places